transactionoutbox-jackson/src/main/java/com/gruelbox/transactionoutbox/jackson/JacksonInvocationSerializer.java

```diff
@@ -43,24 +43,20 @@ public void serializeInvocation(Invocation invocation, Writer writer) {
   }
 
   @Override
-  public Invocation deserializeInvocation(Reader reader) {
-    try {
-      // read ahead to check if old style
-      BufferedReader br = new BufferedReader(reader);
-      if (checkForOldSerialization(br)) {
-        if (defaultInvocationSerializer == null) {
-          throw new UnsupportedOperationException(
-              "Can't deserialize GSON-format tasks without a "
-                  + DefaultInvocationSerializer.class.getSimpleName()
-                  + ". Supply one when building "
-                  + getClass().getSimpleName());
-        }
-        return defaultInvocationSerializer.deserializeInvocation(br);
+  public Invocation deserializeInvocation(Reader reader) throws IOException {
```

Comment:
Does this same problem affect `DefaultInvocationSerializer`?

Reply:
According to JavaDoc it could so I applied the same pattern.